sig-security/self-assessment.md

```diff
@@ -0,0 +1,228 @@
+
+Since joining CNCF, KubeEdge has attracted more than [1000+ Contributors](https://kubeedge.devstats.cncf.io/d/18/overall-project-statistics-table?orgId=1) from 80+ different Organizations with 3,900+ Commits; got 5,600+ Stars on Github and 1,600+ Forks. It has been adopted by China Mobile, China Telecom, Raisecom, WoCloud, Xinghai IoT, KubeSphere, HUAWEI CLOUD, Harmony Cloud, DaoCloud, SAIC Motor, SF Express, etc.
+
+### Background
+
+As 5G and AI technologies grow, enterprises are increasingly demanding intelligent upgrade, and the application scenarios of edge computing are becoming more and more extensive. It is challenging to realize unified management and control of edge computing resources and cloud-edge synergy, such as how to run intelligent applications and algorithms on edge devices with limited resources(e.g. cameras and drones), how to solve the problems caused by the access of mass heterogeneous edge devices in intelligent transportation and intelligent energy, and how to ensure high reliability of services in off line scenarios.
+
+KubeEdge provides solutions for cloud-edge synergy and has been widely adopted in industries including transportation, energy, Internet, CDN, Manufacturing, Smart Campus, etc. KubeEdge provides:
+
+- Seamless Cloud-Edge Communication for both metadata and data.
+- Edge Autonomy: Autonomous operation of Edge even during disconnection from cloud.
+- Low Resource Readiness: KubeEdge can work in constrained resource situations (low memory, low bandwidth, low compute).
+- Simplified Device Communication: Easy communication between application and devices for IOT and IIOT.
+
+### Goal
+
+The main goals of KubeEdge are as follows:
+
+- Building an open edge computing platform with cloud native technologies.
+- Helping users extending their business architecture, applications, services, etc. from cloud to edge but ensuring same user experience.
+- Implementing extensible architecture based on Kubernetes.
+- Integrating with CNCF projects, including (but not limited to) containerd, cri-o, Prometheus, Envoy, etc.
+- Seamlessly develop, deploy, and run complex workloads at the edge with optimized resources.
+
+### Non-goals
+
+The scenarios solved by KubeEdge do not include the problems of getting through the underlying physical network of edge nodes and ensuring the reliability communication of cloud side network infrastructure. However, for the scenarios where edge network is off-line, KubeEdge provides the autonomy of edge off-line nodes.
+
+### Project & Design
+
+The following diagram shows the logical architecture for KubeEdge: 
+
+<img src="assets/kubeedge_overview.png">
+
+KubeEdge consists of below major components:
+
+- CloudCore (in the cloud)

```

Comment:
in the cloud or on the cloud? im not sure.

Reply:
Both can be used as far as I know.